applications/mne_x/plugins/mne/mne.cpp

```diff
@@ -111,7 +111,7 @@ void MNE::init()
     // Inits
     m_pFwd = MNEForwardSolution::SPtr(new MNEForwardSolution(m_qFileFwdSolution));
     m_pAnnotationSet = AnnotationSet::SPtr(new AnnotationSet(m_sAtlasDir+"/lh.aparc.a2009s.annot", m_sAtlasDir+"/rh.aparc.a2009s.annot"));
```

Comment:
Consider using the subject based initialization: AnnotationSet annotationSet("sample", 2, "aparc.a2009s", "./MNE-sample-data/subjects");